App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/TalentFeed/TalentFeed.jsx

```diff
@@ -11,44 +11,101 @@ export default class TalentFeed extends React.Component {
     constructor(props) {
         super(props);
 
-        //let loader = loaderData
-        //loader.allowedUsers.push("Employer")
-        //loader.allowedUsers.push("Recruiter")
+        let loader = loaderData
+        loader.allowedUsers.push("Employer")
+        loader.allowedUsers.push("Recruiter")
 
-        //this.state = {
-        //    loadNumber: 5,
-        //    loadPosition: 0,
-        //    feedData: [],
-        //    watchlist: [],
-        //    loaderData: loader,
-        //    loadingFeedData: false,
-        //    companyDetails: null
-        //}
-
-        //this.init = this.init.bind(this);
+        this.state = {
+            loadNumber: 5,            
+            loadPosition: 1179,
+            feedData: [],
+            watchlist: [],
+            loaderData: loader,
+            loadingFeedData: true,
+            companyDetails: null
+        }
 
+        this.init = this.init.bind(this);
+        this.loadFeedData = this.loadFeedData.bind(this);
+        this.handleScroll = this.handleScroll.bind(this);
     };
 
-    //init() {
-    //    let loader = TalentUtil.deepCopy(this.state.loaderData)
-    //    loader.isLoading = false
+    init() {
+        let loaderData = TalentUtil.deepCopy(this.state.loaderData)
+        loaderData.isLoading = false;
+        this.setState({ loaderData });
+        this.loadFeedData();
+    }
 
+    handleScroll() {
+        const win = $(window);
+        if ((($(document).height() - win.height()) == Math.round(win.scrollTop())) || ($(document).height() - win.height()) - Math.round(win.scrollTop()) == 1) {
+            this.setState({ loadingFeedData: true })
+            this.loadFeedData()
+        }
+    };
 
-    //}
+    componentDidMount() {
+        window.addEventListener('scroll', this.handleScroll);
+        this.init()
+    };
 
-    //componentDidMount() {
-    //    window.addEventListener('scroll', this.handleScroll);
-    //    this.init()
-    //};
+    loadFeedData() {
+        const { feedData, loadPosition, loadNumber } = this.state;
+
+        var cookies = Cookies.get('talentAuthToken');
```

Comment:
Always check when to use **let** and **const** for ES6

Reply:
Changed both to use let instead